woocommerce/woocommerce
# move product list suggestion to the bottom of products list

### All Submissions:


### Changes proposed in this Pull Request:
Positions the product-list table suggestion at the bottom of the list, so it's less intrusive.

Also fixes a minor issue – products list suggestions were getting displayed on mobile. They now will be correctly hidden on smaller screens.

### How to test the changes in this Pull Request:
1. Checkout this branch and get it set up as the WooCommerce plugin in your development environment.
1. Log in as a store administrator and view suggestions in the products list.
1. The suggestion should be displayed at the bottom of the list (previously was displayed after 3rd product).
